static/js/components/widgets/MarkdownEditor.tsx

```diff
@@ -67,7 +71,7 @@ export default function MarkdownEditor(props: Props): JSX.Element {
         editor.current.editing.view.focus()
       }
     },
-    [editor]
+    []
```

Comment:
This is unrelated to everything else. But `editor` is a ref—its reference never changes. So there's no point in including it as a dependency.  (And React's smart linting rules will not complain if we remove it)